pages/common/mongorestore.md

```diff
@@ -0,0 +1,19 @@
+# mongorestore
+
+> Utility to import a collection or database from a binary dump into a MongoDB instance.
+
+- Import a bson data dump from a folder to a mongodb instance:
+
+`mongorestore --db {{database_name}} {{path/to/folder}}`
+
+- Import a bson data dump from a folder to a given mongodb database host running at a given port with user authentication; user will be prompted for password:
+
+`mongorestore --host {{database_host:port}}  --db {{database_name}} --username {{username}} {{path/to/folder}} --password`
```

Comment:
Its true that multiple database can run on the same host, however one port can be bound to only one service (database or any other service accessible via network interface). Typically when someone tries to access a database hosted on a remote machine, and this is the typical usecase. The previous command omits the host and port options, and this works locally on the same machine where database is running. I hope these two combined together covers basic use cases.